### ISSUE TITLE
Error: inject() must be called from an injection context

### ISSUE THREAD

**Issue by tmmschmit** [BuilderIO/builder#203]
Hi, i trying to use BuilderIo on an Angular 9.1.0 app but as soon as I import the module :

```
    BuilderModule.forRoot('...'),
```

I get this error

```
Error: inject() must be called from an injection context
    Angular 23
main.ts:12:24
    ts main.ts:12
    invoke zone-evergreen.js:364
    run zone-evergreen.js:123
    scheduleResolveOrReject zone-evergreen.js:857
    invokeTask zone-evergreen.js:399
    runTask zone-evergreen.js:167
    drainMicroTaskQueue zone-evergreen.js:569
```

Any idea? Thanks!

**Comment by ca136** [comment]
Hi @tmmschmit, do you mind sharing a codesandbox, plunkr or repo with steps to reproduce?

**Comment by tmmschmit** [comment]
Hi, thanks for your answer. I'm sorry, I cannot share part of the project. I've tried to reproduce the bug in another project, but didn't manage to success yet. :thinking: 

I created a new project, with the same config. And lazy loaded the modules from `app-routing` as we do in the project... but it's not this...

About the lazy load, without, I haven't the error in my project, but the component doesn't seem to load at all. :thinking: 

I will come back when I find a way to reproduce the bug.
